README.md

```diff
@@ -108,7 +108,7 @@ Options:
         extension.
         Default: none
   --node-nicknames-from <prop>
-        pactl property to use for node names, unless overriden by
+        pactl property to use for node names, unless overridden by
```

Comment:
This is actually from the `--help` message in the script. Could you fix it there as well?

Reply:
This one: https://github.com/marioortizmanero/polybar-pulseaudio-control/blob/10b08bcf57f7c7007eebab2d739534b1e1da0c81/pulseaudio-control.bash#L461